client/i18n/locales/german/meta-tags.json

```
@@ -24,9 +24,9 @@
     "front",
     "back",
     "end",
-    "learn",
+    "lernen",
     "tutorial",
-    "programming"
+    "programmierung"
   ],
-  "youre-unsubscribed": "You have been unsubscribed"
-}
+  "youre-unsubscribed": "Du wurdest abgemeldet"
+}
```

```suggestion
}

```
do not remove the last new line